e2e/libs/instancemanager/instancemanager.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+from node import Node
+
+from utility.utility import get_longhorn_client
+from utility.utility import get_retry_count_and_interval
+from utility.utility import logging
+
+
+class InstanceManager:
+
+    def __init__(self):
+        self.node = Node()
+
+    def wait_for_all_instance_manager_running(self):
+        longhorn_client = get_longhorn_client()
+        worker_nodes = self.node.list_node_names_by_role("worker")
```

Comment:
Why limit to `worker_nodes`? Why not just check all instance managers ready? Why need to check `if len(instance_manager_map) == len(worker_nodes): break`?